src/Symfony/Component/Messenger/Attribute/AsMessageHandler.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Attribute;
+
+/**
+ * Service tag to autoconfigure message handlers.
+ *
+ * @author Maxim Dovydenok <dovydenok.maxim@gmail.com>
+ */
+#[\Attribute(\Attribute::TARGET_CLASS)]
+class AsMessageHandler
+{
```

Comment:
It's also possible to configure an optional `bus` and `from_transport` on the `messenger.message_handler`. Maybe that should be added to the attribute as well?

https://symfony.com/doc/current/messenger.html#manually-configuring-handlers